src/components/MyBundle/MyBundleDetail/MyBundleDetail.style.ts

```diff
@@ -1,10 +1,27 @@
 import styled from 'styled-components';
 
+import { MOBILE } from '@/constants';
 import { Col } from '@/styles/globalStyles';
 
-export const Container = styled(Col)`
+export const Container = styled.section<{ $isBundleSelected: boolean }>`
+  width: ${({ $isBundleSelected }) => ($isBundleSelected ? '60%' : '80%')};
+  height: 100%;
+  padding-top: 1rem;
+  box-sizing: border-box;
+
+  transition: all 1s ease;
+
+  @media screen and (max-width: ${MOBILE}px) {
+    width: 100%;
+  }
+`;
+
+export const InnerContainer = styled(Col)`
   width: 100%;
+  height: 100%;
   padding: 2rem;
```

Comment:
```suggestion
  padding: 1rem 2rem;
```
정도로 상하 여백은 조금 줄여도 좋을 것 같습니다!